interfaces/builtin/repowerd.go

```diff
@@ -0,0 +1,335 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var repowerdPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the repowerd service. Reserved because this
+#  gives privileged access to the system.
+# Usage: reserved
+
+# HW alarms
+/dev/alarm rw,
+
+# Suspend
+/sys/power/** rw,
+
+# Backlight
+/sys/**/brightness rw,
```

Comment:
Well, we always can, but ideally we wouldn't. Given that the slot availability will be pretty restricted nevertheless, since it's a permanent one, we might allow it in. @jdstrand: any ideas?